SPECS/pcre/pcre.spec

```
@@ -53,9 +58,9 @@ make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-mv -v %{buildroot}/usr/lib/libpcre.so.* %{buildroot}/lib &&
-ln -sfv ../../lib/$(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}%{_lib}/libpcre.so
-ln -sfv $(readlink %{buildroot}/usr/lib/libpcre.so) %{buildroot}%{_lib}/libpcre.so.0
+mv -v %{buildroot}%{_lib}/libpcre.so.* %{buildroot}/lib &&
+ln -sfv ../../lib/$(readlink %{buildroot}%{_lib}/libpcre.so) %{buildroot}%{_lib}/libpcre.so
+ln -sfv $(readlink %{buildroot}%{_lib}/libpcre.so) %{buildroot}%{_lib}/libpcre.so.0
```

We have to watch out for these. For 1.0 this is fine but the `%{_lib}` macro has changed in 2.0, so this would break during migration.

```suggestion
mv -v %{buildroot}%{_libdir}/libpcre.so.* %{buildroot}/lib &&
ln -sfv ../../lib/$(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_lib}/libpcre.so
ln -sfv $(readlink %{buildroot}%{_libdir}/libpcre.so) %{buildroot}%{_lib}/libpcre.so.0
```

It's a very tribal-knowledge thing, sorry for the trouble.